.github/workflows/self-test.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Statick
+
+on: # NOLINT
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          path: "statick"
```

Comment:
This tripped me up for longer than it should have.  I didn't realize that the checkout action doesn't result in a named repository.  It just places the contents of the repo directly under `/github/workflow`.   Statick appeared to run fine, except that it would think the package name was "workspace" instead of "statick," and a couple of our self-check exceptions specify package name "statick."    Took me a while to figure that out - maybe we should have statick report the package name it's using at the INFO level?    So this `path` parameter tells checkout to place the contents of the repo under a directory called `statick.`.